tutorials/jdbcIngest/src/main/resources/META-INF/pollerProperties.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
```

Comment:
Is the name of this XML file correct ? From the documentation at
http://docs.datatorrent.com/application_packages/
it seems that application-specific properties files should be named properties-{appName}.xml

Please run the example on a cluster and double check that the README.md has complete and accurate instructions on what the user needs to do to successfully run the application on a cluster -- we want the user experience to be as smooth and painless as possible when running these examples both in local mode and on a cluster.